lib/lib.js

```diff
@@ -74,7 +74,12 @@ function maybeTestForPlusOne(config, comment) {
   return /\+1/.test(comment);
 }
 
+exports.formatReviewedByMetaData = formatReviewedByMetaData;
 function formatReviewedByMetaData(reviewer) {
+  if (!reviewer.email) {
```

Comment:
thanks, applied. 

i am not super convinced as i usually prefer early returns. personally not worried about duplication in this case. also not worried about the abstraction either.

i changed the code according to the request can you explain why you prefer it in this way, i want to understand it better and learn.